internal/tools/securitypolicy/helpers/helpers.go

```diff
@@ -92,6 +92,19 @@ func ParseWorkingDirFromImage(img v1.Image) (string, error) {
 	return "/", nil
 }
 
+// ParseCommandFromImage inspects the image and returns the command args, which
+// is a combination of ENTRYPOINT and CMD Docker directives.
+func ParseCommandFromImage(img v1.Image) ([]string, error) {
+	imgConfig, err := img.ConfigFile()
+	if err != nil {
+		return nil, err
+	}
+
+	cmdArgs := imgConfig.Config.Entrypoint
+	cmdArgs = append(cmdArgs, imgConfig.Config.Cmd...)
+	return cmdArgs, nil
+}
+
 // PolicyContainersFromConfigs returns a slice of securitypolicy.Container generated
 // from a slice of securitypolicy.ContainerConfig's
 func PolicyContainersFromConfigs(containerConfigs []securitypolicy.ContainerConfig) ([]*securitypolicy.Container, error) {
```

Comment:
I have not followed all of these CR's here but is the ask to drop Entrypoint if the `containerConfig` policy has an override command? Or is it to append that command to the images entrypoint?

Reply:
if command isn't provided as part of the `policy.toml` file (https://github.com/microsoft/hcsshim/tree/master/internal/tools/securitypolicy#example-toml-configuration-file), then we try to read the ENTRYPOINT and CWD from the image, since that's what CRI is going to do and if we don't, we won't be able to enforce the command policy.